Packs/FeedUnit42v2/Integrations/FeedUnit42v2/FeedUnit42v2.py

```diff
@@ -140,9 +142,15 @@ def parse_indicators(indicator_objects: list, feed_tags: Optional[list] = None,
                         }
                     }
 
-                    if "file:hashes.'SHA-256' = '" in indicator_obj['value']:
-                        if ioc_value := extract_ioc_value(indicator_obj['value']):
+                    if "file:hashes.'SHA-256' = '" in pattern:
+                        if ioc_value := extract_ioc_value(pattern):
                             indicator_obj['value'] = ioc_value
+                            
+                        if ioc_value == raw_name:
+                            continue
```

Comment:
Please add a debug message that the indicator was dropped

Reply:
i remove this part